spec/index.html

```diff
@@ -582,15 +582,16 @@ <h3>IRIs</h3>
     <p class="note">For convenience, a complete [[ABNF]] grammar
       from [[RFC3987]] is provided in <a href="#iri-abnf" class="sectionRef"></a>.</p>
 
-    <p>IRIs in the RDF abstract syntax MUST be absolute,
+    <p>IRIs in the RDF abstract syntax MUST be <a data-cite="RFC3986#section-5">fully resolved</a>
+      per [[RFC3986]],
       and MAY contain a fragment identifier.</p>
 
     <p><dfn>IRI equality</dfn>:
       Two IRIs are the same if and only if they compare as identical when
       considered as character strings, as in Simple String Comparison in
       <a data-cite="rfc3987#section-5.3.1">section 5.3.1</a>
       of [[!RFC3987]].
-      (This is done in the abstract syntax, so the IRIs are absolute
+      (This is done in the abstract syntax, so the IRIs are fully resolved
```

Comment:
```suggestion
      (This is done in the abstract syntax, so the IRIs are resolved
```
I don't think there is a "partially resolved".